int32 INFO=0
int32 WARN=1
int32 ERROR=2
# Status type.  One of INFO, WARN, ERROR.
int32 type
# Status message.
string status
